lib/pages/chat/chat.dart

```diff
@@ -363,16 +431,20 @@ class ChatController extends State<Chat>
     context.pop();
   }
 
+  void _jumpToLastMessageFullyRead() async {
+    await getTimeline();
+    if (room == null || room!.notificationCount <= 0) return;
+    final lastReadEvent = room!.fullyRead;
+    await Future.delayed(waitForJumpToFirstUnreadEvent);
+    _isJumpingToLastReadEvent = true;
+    scrollToEventId(lastReadEvent);
```

Comment:
scrollToEventId is a future, should wait for it, instead `waitForJumpToFirstUnreadEvent`

Reply:
It's a void func, not future